pkg/pubsub/adapter/converters/storage.go

```diff
@@ -93,7 +94,8 @@ func convertStorage(ctx context.Context, msg *cepubsub.Message, sendMode ModeTyp
 	// Attributes are extensions.
 	if msg.Attributes != nil && len(msg.Attributes) > 0 {
 		for k, v := range msg.Attributes {
-			event.SetExtension(k, v)
+			// V1 attributes MUST consist of lower-case letters ('a' to 'z') or digits ('0' to '9').
+			event.SetExtension(strings.ToLower(k), v)
```

Comment:
Is there a unit test for this lowercasing?

Reply:
same as above... If you want them now, I can go for it... Otherwise in the several follow ups we need to do in this repo

Reply:
done!